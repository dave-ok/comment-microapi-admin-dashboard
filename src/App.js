import React from 'react';
import { Admin, Resource } from 'react-admin';
import Dashboard from './components/dashboard/Dashboard';
import authProvider from './services/authProvider';
import adminProvider from './services/adminProvider';
import UserIcon from '@material-ui/icons/Group';
import { AdminList, AdminEdit, AdminCreate } from './components/admin/Admins';

function App() {
  return (
    <Admin
      dashboard={Dashboard}
      authProvider={authProvider}
      dataProvider={adminProvider}
    >
      <Resource
        name="msadmins"
        list={AdminList}
        edit={AdminEdit}
        create={AdminCreate}
        icon={UserIcon}
      />
    </Admin>
  );
}

export default App;
