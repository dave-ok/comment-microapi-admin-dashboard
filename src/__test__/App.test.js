import * as React from "react";
import { shallow } from "enzyme";
import App from "../App";

test("renders Dashboard", () => {
  const app = shallow(<App />);
  expect(app).toMatchSnapshot();
});
