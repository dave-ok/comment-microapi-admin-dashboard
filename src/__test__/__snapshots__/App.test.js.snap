// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Dashboard 1`] = `ShallowWrapper {}`;
